### DISCUSSION TITLE
How to downgrade firmware

### DISCUSSION THREAD

**Discussion by SleepingSnorlax-99** [roleoroleo/yi-hack-Allwinner-v2#578] [General]
Hi,

i've a 1080 Outdoor cam YHS.3020 (IFUS) and it's on 11+ firmware, I know this isn't supported yet but I wondered if there was a way I could downgrade to a firmware version compatible with this hack please?

thanks in advance

**Comment by roleoroleo** [comment]
Follow the unbrick wiki page.
It's the same procedure.
